libfuzzer-dotnet-windows.cc

```diff
@@ -153,40 +171,39 @@ FUZZ_EXPORT int __cdecl LLVMFuzzerInitialize(int *argc, char ***argv)
     hMemFile = CreateFileMapping(INVALID_HANDLE_VALUE, NULL, PAGE_READWRITE, 0, MAP_SIZE + DATA_SIZE, sharedMemName);
     if (hMemFile == NULL)
     {
-        die_sys("CreateFileMapping() failed");
+        die_lasterror("CreateFileMapping() failed");
     }
 
     atexit(close_shm);
     pBuf = MapViewOfFile(hMemFile, FILE_MAP_ALL_ACCESS, 0, 0, MAP_SIZE + DATA_SIZE);
     if (pBuf == NULL)
     {
-        CloseHandle(hMemFile);
```

Comment:
Removed this `CloseHandle` since it will (potentially) overwrite `GetLastError`, and in any case we're about to exit the process.